### PR TITLE
adding my new project

### PR DESCRIPTION
**Pull request by skekre98**
👋 I think this project would be a great addition to Up for Grabs

Let me know if you have any feedback on my template!

### CODE REVIEW

File: _data/projects/NbaSearch.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  - html
+  - css
+  - flask
+  - pandas
+  - sklearn
+  - bs4
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/skekre98/NBA-Search/issues?q=is%3Aissue+is%3Aopen+label%3Aup-for-grabs
+  name: good first issue
```
Comment (shiftkey) [inline _data/projects/NbaSearch.yml]: Currently we don't support multiple listings in the single file - #244 is our tracking issue for that.

I'd recommend choosing one label to list here to keep it simple - up to you which one you'd prefer to see.

**Review by shiftkey — CHANGES_REQUESTED** [review]
Thanks for the contribution @skekre98!

Unfortunately I can't merge this in as-is, but I think it's close!

File: _data/projects/NbaSearch.yml (hunk 1 above)
Comment (skekre98) [reply _data/projects/NbaSearch.yml]: @shiftkey removed the extra tag. Let me know if there is anything else that needs updating 👍

File: _data/projects/NbaSearch.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  - javascript
+  - html
+  - css
+  - flask
+  - pandas
+  - sklearn
+  - bs4
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/skekre98/NBA-Search/issues?q=is%3Aissue+is%3Aopen+label%3Aup-for-grabs
```
Comment (shiftkey) [inline _data/projects/NbaSearch.yml]: Some tidy up for the GitHub integration:

```suggestion
  link: https://github.com/skekre98/NBA-Search/labels/up-for-grabs
```

**Review by shiftkey — APPROVED** [review]
Thanks for the contribution @skekre98!

**Pull request merged**